src/System.ServiceModel.Security/pkg/System.ServiceModel.Security.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>wp8</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\ref\System.ServiceModel.Security.csproj">
-      <SupportedFramework>net46;netcore50;dnxcore50</SupportedFramework>
+      <SupportedFramework>net45;netcore45;dnxcore50</SupportedFramework>
```

Comment:
Build errors will tell you what to do, also you can examine the targeting pack XML for the past profiles.